mesonbuild/msetup.py

```diff
@@ -51,6 +51,9 @@ def add_arguments(parser):
                         help='Set options and reconfigure the project. Useful when new ' +
                              'options have been added to the project and the default value ' +
                              'is not working.')
+    parser.add_argument('--clearcache', action='store_true',
+                        help='Does the same as --reconfigure and additionally clears' +
+                             'the internal meson cache (dependencies, etc.).')
```

Comment:
This should say that it clears the cache first and does reconfigure only afterwards. Just to be sure.